client/snapcraft.yaml

```
@@ -1,11 +1,22 @@
 name: hwctl
 adopt-info: hwctl
+license: GPL-3.0-only
 summary: Canonical Certified Hardware API Client
 description: |
-  hwctl is a hwlib backed CLI tool for collecting and sending hardware
-  information to hwapi to find testing status of device models
+  `hwctl` is a `hwlib` backed CLI tool for collecting and sending hardware
+  information to a Hardware API server to find testing status of device models.
```

```suggestion
  information to a Hardware API server to find certification status of device models.
```

At least this is what we’re saying in a Debian package description

---

Let's be consistent then, I've changed it